### ISSUE TITLE
Sometimes *.output files are not deleted automatically. 

### ISSUE THREAD

**Issue by yunikkk** [gojuno/composer#14]
Can't figure out the trigger yet, but sometimes *.output is not deleted. 
It contained 
`
Script started on Tue Apr 11 15:01:33 2017
command: /Users/dmitryyunitsky/Library/Android/sdk/platform-tools/adb devices

Script started on Tue Apr 11 15:01:33 2017
command: /Users/dmitryyunitsky/Library/Android/sdk/platform-tools/adb devices
List of devices attached

Script started on Tue Apr 11 15:01:33 2017
command: /Users/dmitryyunitsky/Library/Android/sdk/platform-tools/adb devices

Script started on Tue Apr 11 15:01:33 2017
command: /Users/dmitryyunitsky/Library/Android/sdk/platform-tools/adb devices
List of devices attached

Script started on Tue Apr 11 15:01:33 2017
command: /Users/dmitryyunitsky/Library/Android/sdk/platform-tools/adb devices
List of devices attached
emulator-5554   device
Script done on Tue Apr 11 15:01:33 2017

Script started on Tue Apr 11 15:01:33 2017
command: /Users/dmitryyunitsky/Library/Android/sdk/platform-tools/adb devices
`

**Comment by koral--** [comment]
I've also seen this issue.
At a glance, there is `File.deleteOnExit()` https://github.com/gojuno/commander/blob/4fea273b24007c3787bbbad538b6e8e1dbde3972/os/src/main/kotlin/com/gojuno/commander/os/Processes.kt#L115
so file won't be deleted if JVM process terminates abnormally.

**Comment by artem-zinnatullin** [comment]
Yep, that's expected, nothing we can really do here I guess unless it's a normal case when we expect files to be deleted. 

I think with proper redirection to a folder instead of current working dir this will is fine.
